tags/outputs.tf

```diff
@@ -1,9 +1,9 @@
 output "ids" {
-  description = "The id of the tag."
+  description = "The list of id of the new tags."
```

Comment:
Please use: The ids of the created tags. Does not include the ids of pre-existing tags.